docs/README-utilities.md

```diff
@@ -0,0 +1,38 @@
+# Executable utilities included in this gem
+
+## check_metric_limits.rb
```

Comment:
Just to be clear, I didn't create the check_metric_limits.rb utility. I just found it in the bin directory and figured I would document it along with our new show_running_yang utility.